modules/ROOT/pages/styleguide.adoc

```diff
@@ -472,7 +472,7 @@ RETURN 'Cypher\'s a nice language', "Mats' quote: \"statement\""
 RETURN "Cypher's a nice language", 'Mats\' quote: "statement"'
 ----
 
-* Avoid having to use back-ticks to escape characters and keywords.
+* Avoid having to use backticks to quote characters and keywords.
```

Comment:
```suggestion
* Avoid using characters and keywords that require the input to be quoted.
```
Feels a little bit more actionable than "avoid having the db force a rule on you"